integration_tests/sdk/resources_test.py

```diff
@@ -110,3 +110,58 @@ def fn_expect_failure():
         engine=engine,
         expect_success=False,
     )
+
+
+@pytest.mark.enable_only_for_engine_type(ServiceType.K8S)
```

Comment:
I think --gpu should be a another flag that we enable similarly, eg: `@pytest.mark.require_gpu()`, can we file a task for this?

Reply:
https://linear.app/aqueducthq/issue/ENG-2013/add-gpu-flag-to-sdk-integration-tests